docs/user/adding-glean-to-your-project.md

```diff
@@ -212,7 +212,9 @@ Follow these steps to automatically run the parser at build time:
    > **Important**: The parser generates one file per category.
    > If you are unsure which files those are, skip this step and add the files after they are generated for the first time.
 
-5. If you are using Git, add the following line to your `.gitignore` file:
+5. Still on your application targets' Build Phases settings tab, add the files from the `/Generated` directory to the "Compile Sources" build phase.
```

Comment:
Ooooh, right.
The way it's currently done for Glean itself and also the sample app is that the project has the `Generated` folder and the files were added to the project once, at which point they got added to "Compile Sources".

Reply:
Yup, I noticed this when creating the framework and made sure to add it in the docs.  Other than that, the process for including in a framework is the same as an app, minus the init.